app/components/project_submissions/title_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if url.present? %>
-  <%= link_to title, url, class: 'truncate max-w-xs lgs:max-w-lg font-medium text-lg break-words hover:text-gray-800', data: { turbo_frame: '_top' } %>
+  <%= link_to title, url, class: 'truncate max-w-xs lgs:max-w-lg font-medium text-lg break-words hover:text-gray-900', data: { turbo_frame: '_top' } %>
```

Comment:
just a note that the below PR that's ready for merging still has `hover:text-gray-800` and an added `dark:hover`: 

https://github.com/TheOdinProject/theodinproject/pull/4016/ 

Noting this since this PR for the `likes` doesn't describe the change you've made here as far as I can see

Reply:
Great catch 💪 reverted to 800.